.github/workflows/release.yml

```
@@ -76,4 +76,10 @@ jobs:
     - name: Run y.sh cargo build
       run: |
         EMBED_LTO_BITCODE=1 CHANNEL="release" ./y.sh cargo build --release --manifest-path tests/hello-world/Cargo.toml
-        # TODO: grep the asm output for "call my_func" and fail if it is found.
+        asm=$(objdump -dj .text tests/hello-world/target/release/hello_world)
+        if echo asm | grep -q "call .*mylib.*my_func" ; then 
```

It's probably more efficient to do:
```suggestion
        call_found=$(objdump -dj .text tests/hello-world/target/release/hello_world | grep -q "call .*mylib.*my_func")
        if call_found then 
```